[unicorn/import-style] false positive for importing some TypeScript interfaces

<!-- ✨ Thanks for reporting a bug! ➡️ Please don't ignore this template -->

<!-- 1️⃣ Explain here what's wrong -->

When importing the TypeScript interface `ChalkInstance` from `chalk`:

```typescript
import { ChalkInstance } from "chalk";
```

`eslint-plugin-unicorn` gives the following error:

```
/Users/leonzalion/projects/playground/chalk/test.ts
  1:1  error  Use default import for module `chalk`  unicorn/import-style

✖ 1 problem (1 error, 0 warnings)
```

However, there is no default import for the `ChalkInstance` type.

My guess is that it has something to do with the following lines in the chalk type definitions:

```typescript
declare const chalk: ChalkInstance;
export default chalk;
```

<!-- 2️⃣ Specify which rule is buggy here and in the title -->

<!-- 3️⃣ Add some examples where the issue appears -->
**Repro:** https://github.com/leonzalion/unicorn-import-style-bug

--- a/rules/import-style.js
+++ b/rules/import-style.js
@@ -7,6 +7,10 @@ const messages = {
 };
 
 const getActualImportDeclarationStyles = importDeclaration => {
+	if (importDeclaration.importKind === 'type') {
+		return [];
+	}
+
 	const {specifiers} = importDeclaration;
 
 	if (specifiers.length === 0) {
@@ -27,6 +31,10 @@ const getActualImportDeclarationStyles = importDeclaration => {
 		}
 
 		if (specifier.type === 'ImportSpecifier') {
+			if (specifier.importKind === 'type') {
+				continue;
+			}
+
 			if (specifier.imported.type === 'Identifier' && specifier.imported.name === 'default') {
 				styles.add('default');
 				continue;

--- a/test/import-style.js
+++ b/test/import-style.js
@@ -714,6 +714,20 @@ test.babel({
 	].map(test => addDefaultOptions(test)),
 });
 
+test.typescript({
+	valid: [
+		{
+			code: 'import {type ChalkInstance} from \'chalk\'',
+			options: [],
+		},
+		{
+			code: 'import type {ChalkInstance} from \'chalk\'',
+			options: [],
+		},
+	],
+	invalid: [],
+});
+
 test.snapshot({
 	valid: [
 		'let a',

